__tests__/components/RequestForm.test.tsx

```
@@ -0,0 +1,265 @@
+    codeLoading: false,
+    selectedCodeLanguage: 'curl',
+    setSelectedCodeLanguage: jest.fn(),
+    error: null,
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterAll(() => {
+    console.error = console.error;
```

Biome (1.9.4)

[error] error is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)